.mocharc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   timeout: 30000,
   ui: 'bdd',
   extension: ['js', 'spec.cjs', 'spec.mjs', 'spec.ts', 'spec.mts'],
-  require: ['mocha-expect-snapshot/old-format'],
+  require: ['mocha-expect-snapshot/old-format', 'ts-node/register'],
```

Comment:
Since we are using esbuild-kit/esm-loader for esm, probably we should use esbuild-kit/cjs-loader for commonjs.
https://github.com/esbuild-kit/cjs-loader

Reply:
I added the loader but with that mocha doesn't work anymore. The error message is 

> ✖ ERROR: /home/runner/work/generator-jhipster/generator-jhipster/node_modules/@cspotcode/source-map-support/source-map-support.js:49
>   var sharedData = this[sharedDataKey];
>                    ^
> 
> TypeError: Cannot read properties of undefined (reading 'Symbol(source-map-support/sharedData)')

See here: https://github.com/jhipster/generator-jhipster/actions/runs/3169199634/jobs/5160937434

Since mocha works without this package I removed it for now.